tests/test_variables_script.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash  
```

Comment:
I added the script to add another layer of test to help the user understand in case of a problem. If the user add another variable for a module that is necessary for the test template to run and didn't add it to the test template, the terraform plan step will fail, but i wanted to make sure that the user will understand what cause the error and what he is missing. I can remove the script if you don't think that it is necessary.